bucket/qtpass.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "1.3.2",
+    "license": "GPL-3.0-only",
+    "homepage": "https://qtpass.org/",
+    "description": "GUI for pass, the standard UNIX password manager.",
+    "url": "https://github.com/IJHack/QtPass/releases/download/v1.3.2/qtpass-1.3.2.exe",
+    "hash": "0fb9e68ce7e52e614ebb29dd9fc0b06b24f4262e9509090853e2c2991c171e72",
+    "innosetup": true,
```

https://github.com/IJHack/QtPass/blob/d7d7443ad8e607ac0f483984bf8814f6f4aa83d0/src/qtpasssettings.cpp#L18-L30

```suggestion
    "innosetup": true,
    "pre_install": [
        "ensure \"$persist_dir\" | Out-Null",
        "if (Test-Path \"$persist_dir\\qtpass.ini\") {",
        "    Copy-Item \"$persist_dir\\qtpass.ini\" \"$dir\" -ErrorAction SilentlyContinue",
        "} else {",
        "    New-Item \"$dir\\qtpass.ini\" | Out-Null",
        "}"
    ],
    "uninstaller": {
        "script": "Copy-Item \"$dir\\qtpass.ini\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
    },
```